installation/sql/mysql/base.sql

```diff
@@ -1126,7 +1132,8 @@ CREATE TABLE IF NOT EXISTS `#__workflow_stages` (
 --
 
 INSERT INTO `#__workflow_stages` (`id`, `asset_id`, `ordering`, `workflow_id`, `published`, `title`, `description`, `default`) VALUES
-(1, 0, 1, 1, 1, 'COM_WORKFLOW_BASIC_STAGE', '', 1);
+(1, 0, 1, 1, 1, 'COM_WORKFLOW_BASIC_STAGE', '', 1),
+(2, 18, 1, 2, 1, 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES', '', 1);
```

Comment:
```suggestion
(2, 90, 1, 2, 1, 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES', '', 1);
```
I would expect asset ID 90 and not 18 here, lice you did it for PostgreSQL already right

And is the title 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES' of the workflow the right description for the stage here?

Reply:
You are right, I will push another commit to fix the incorrect string for the name. It should be the default workflow stage name